AB_environments/AB_sample.conda.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   - fsspec ==2024.6.1
   - s3fs ==2024.6.1
   - gcsfs ==2024.6.1
-  - pyarrow ==17.0.0
+  - pyarrow ==15.0.0
```

Comment:
why?

Reply:
to keep this in sync with `environment.yml`

Reply:
Ah sorry, unrelated change...